gzip_logs_tools/log_compressor/log_compressor.c

```diff
@@ -688,7 +689,7 @@ init_state(state_t* state, const char* input_owner, char *args)
 	}
 	
 	colon_pos = strchr(args, ':');
-	if (colon_pos == NULL || colon_pos - args >= sizeof(input_path))
+	if (colon_pos == NULL || colon_pos - args >=  (long) sizeof(input_path)) // safe to cast sideof(input_path)<4096
```

Comment:
since colon_pos must be >= args, i'd prefer (size_t) (colon_pos - args) instead

Reply:
👍